.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: 'ESLint check'
+on: [pull_request]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gnosis/safe-react-eslint-plus-action@v4.0.0
```

Comment:
what's the difference between https://github.com/bradennapier/eslint-plus-action and `gnosis/safe-react-eslint-plus-action`?

Reply:
Basically for this [issue](https://github.com/bradennapier/eslint-plus-action/issues/72) in the lib and also last release was about 2 years ago. Gnosis forked the repo and they are keeping it updated since they are using it for the `safe-react` project

Reply:
A bit of an irony:
<img width="988" alt="image" src="https://user-images.githubusercontent.com/2352112/161446205-5292a696-32b6-451f-8809-dfaf46c37e28.png">